informative README

coolharsh55 opened the pull request:
The README file gives a good overview and information about the project. It should contain information that describes the project, it's aim and goals, how to install and run it along with development concerns such as contributions and tests.

Closes #8 informative initial readme for project

Signed-off-by: Harshvardhan Pandit coolharsh55@gmail.com

coolharsh55 committed:
informative README

The README file gives a good overview and information about the project. It should contain information that describes the project, it's aim and goals, how to install and run it along with development concerns such as contributions and tests.

Signed-off-by: Harshvardhan Pandit <coolharsh55@gmail.com>

SanketDG commented on README.md:
```diff
@@ -1,5 +1,45 @@
+Great! Come join our group-talks at slack and fire away all those awesome ideas in your brain.
+
+### Testing?
+Sure! In fact, the more tests you run, the better chance you have at catching bugs! So you can write tests, welcome! If you can run tests and find bugs, still welcome!
```
Add some lines about testing.

The testing dependencies are located at `requirements/testing.txt`

After a pip install from that, one would run `pytest` from that.

coolharsh55 committed:
requirements files for dev, test, and prod

Requirements files for developement, testing, and production using pip-tools. The *.in files contain the dependencies explicitily added to the project. The *.txt files contain the actual dependencies as calculated using pip-tools. Use the *.txt files for installing dependencies.
Closes #12

Signed-off-by: Harshvardhan Pandit <coolharsh55@gmail.com>

coolharsh55 committed:
updated travis settings

Contains comments, configurations for cache, notifications, requirements files. Is configured to send notifications to #repo slack room.
Closes #11

Signed-off-by: Harshvardhan Pandit <coolharsh55@gmail.com>

tobias47n9e commented:
Should we just merge this and add more detail later?
